examples/ansible-vars.yaml

```diff
@@ -1,27 +1,27 @@
+---
 ## Cluster
 PROVIDER: <provider>
 CLUSTER_NAME: <cluster_name>
 NAMESPACE: <namespace>
 
 ## MISC
 DEPLOY: true # Set to false to stop at generating the manifest
-LOADBALANCER_SOURCE_RANGES: ['<cluster_nat_ip>/32']
-
```

Comment:
@thpang, looks like only the quotes were deleted by the linter's recommended change, the relocation of the ## Jump label to line 10 caused the difference engine to show the line as removed and added in another location.

Reply:
@thpang, I think this one can be resolved since that line was not deleted as the diff engine makes it appear. It's easiest to check the full file to confirm.